v2/pkg/engine/plan/planner.go

```diff
@@ -313,8 +316,8 @@ func (p *Planner) printPlanningPaths() {
 	p.debugMessage("Planning paths:")
 	for i, planner := range p.configurationVisitor.planners {
 		fmt.Println("Paths for planner", i+1)
-		fmt.Println("Planner parent path", planner.parentPath)
-		for _, path := range planner.paths {
+		fmt.Println("Planner parent path", planner.ParentPath())
```

Comment:
this Println is only called in debug mode, right?

Reply:
yeah, it is called like this
```
	if p.config.Debug.PrintPlanningPaths {
		p.printPlanningPaths()
	}
```